### PR TITLE
Add designs for choosing between spring or autumn

### PR DESCRIPTION
**Pull request by DominiqueSherwood1**
Branch for the soft-close flow

### CODE REVIEW

File: app/routes/soft-close.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    res.render(`soft-close/${req.params.view}`, { paths: softCloseWizardPaths(req) })
+  })
+
+  router.post([
+    '/soft-close',
+    '/soft-close/:view'
+  ], function (req, res) {
+    const fork = softCloseWizardForks(req)
+    const paths = softCloseWizardPaths(req)
+    console.log(paths)
```
Comment (fofr) [inline app/routes/soft-close.js:21]: You should remove this line now

File: app/utils/soft-close-wizard-paths.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  ]
+
+  return nextAndBackPaths(paths, req)
+}
+
+function softCloseWizardForks (req) {
+  var forks = [
+    // Example fork:
+    // Skip the England question if an answer other
+    // than England is given for where you live
```
Comment (fofr) [inline app/utils/soft-close-wizard-paths.js:21]: These comments can be deleted

File: app/views/notification/sign-up.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {% extends "_wizard-form.html" %}
-{% set title = 'Sign up for alerts' %}
+{% set title = 'Sign up for an alert' %}
```
Comment (fofr) [inline app/views/notification/sign-up.html:2]: Nice 👍

File: app/views/soft-close/alert.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{% extends "_wizard-form.html" %}
+{% set title = 'Registration has temporarily closed.' %}
+
+
+
```
Comment (fofr) [inline app/views/soft-close/alert.html:5]: Probably worth cleaning up the whitespace here

File: app/views/soft-close/chosen.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{% extends "_wizard-form.html" %}
+{% set title = 'Have you agreed with a provider that you will start your NPQ in February?' %}
```
Comment (fofr) [inline app/views/soft-close/chosen.html:2]:
```suggestion
{% set title = 'Have you agreed with a provider that you’ll start your NPQ in February?' %}
```
?

File: app/views/soft-close/no-confirm.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{% extends "_wizard-page.html" %}
+{% block pageNavigation %}{% endblock %}
+{% set title = 'You have chosen not to receive an alert' %}
```
Comment (fofr) [inline app/views/soft-close/no-confirm.html:3]:
```suggestion
{% set title = 'You will not get an alert' %}
```
I wonder if this page should not have the green success panel

Something pared back like this?
Screenshot 2021-12-13 at 10 14 14

File: app/views/soft-close/no-confirm.html [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+
+{% block content %}
+<div class="govuk-grid-row">
+
+
```
Comment (fofr) [inline app/views/soft-close/no-confirm.html:9]: Lots of whitespace here too

**Comment by DominiqueSherwood1** [comment]
01 Agreed with provider question

02 Registration closed

03 Not signed up for alerts

File: app/views/soft-close/test.html [hunk 8]
```diff
@@ -0,0 +1,24 @@
+{% extends "_wizard-form.html" %}
```
Comment (fofr) [inline app/views/soft-close/test.html:1]: I think this file can be deleted

File: app/views/soft-close/chosen.html (hunk 5 above)
Comment (DominiqueSherwood1) [reply app/views/soft-close/chosen.html:2]: Will do and made sure it's quotation marks ;)

File: app/views/soft-close/no-confirm.html (hunk 6 above)
Comment (DominiqueSherwood1) [reply app/views/soft-close/no-confirm.html:3]: Yep, good point!

File: app/views/soft-close/alert.html [hunk 9]
```diff
@@ -0,0 +1,32 @@
+{% extends "_wizard-form.html" %}
+{% set title = 'Registration has temporarily closed.' %}
```
Comment (fofr) [inline app/views/soft-close/alert.html:2]:
```suggestion
{% set title = 'Registration has temporarily closed' %}
```
We do not need full stops in titles here

File: app/views/soft-close/no-confirm.html [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+  <div class="govuk-grid-column-two-thirds">
+    {{ govukPanel({
+      titleHtml: title | noOrphans | safe,
+      html: html,
+      classes: 'govuk-!-margin-bottom-6'
+    }) }}
+    <p>If you change your mind, you can revisit <a href="/start">Register for a National Professional Qualification.</a></p>
```
Comment (fofr) [inline app/views/soft-close/no-confirm.html:16]:
```suggestion
    <p>If you change your mind, you can revisit <a href="/start">Register for a National Professional Qualification</a>.</p>
```
Small tweak to move `.` outside of link

File: app/views/soft-close/no-confirm.html (hunk 6 above)
Comment (DominiqueSherwood1) [reply app/views/soft-close/no-confirm.html:3]: Screenshot 2022-01-13 at 16 02 04
 I'll leave it like this for now, as pairing with James tomorrow

**Comment by DominiqueSherwood1** [comment]
Main changes:

- Content tweaks 
 - Before you start content on /start when the service is fully closed 
 - Taken the email capture (/sign-up) out of the soft close flow and added a conditional reveal if a user says 'yes' to an email alert
 - Will need to check with policy about guidance placement

**Comment by DominiqueSherwood1** [comment]
01-softclose-start
02-softclose-agreed
03-softclose-alert
04-softclose-yes
05-softclose-no
06-closed-start
07-softclose-signup

**Comment by DominiqueSherwood1** [comment]
This merge is to make sure the prototype is up to date. 

There have been content changes for the hard close, so worth keeping consistent before build and the soft-close was never added to the prototype

**Pull request merged**